docs/introduction/what-is-resift.md

```diff
@@ -0,0 +1,35 @@
+---
+id: what-is-resift
+title: What is ReSift?
+sidebar_label: What is ReSift?
+---
+
+Simply put, **ReSift is a state management library for fetches** with the goal of giving your team a capable standard for fetching, storing, and reacting to data.
+
+## Motivation
+
+When you start to develop a React application, you soon realize that there is a lot more than meets the eye regarding data fetching.
+
+With every fetch we need to know:
+
+- If the data request is inflight (so we can show a loading indicator)
+- Where to store the data and how to look it up
+- If it's related to other fetches so they update consistently
+- If the response is an error so we try to recover
+
+These tasks themselves aren't overly complicated but doing them over and over and over again _will_ take time away from your product.
+
+To make matters worse, there's an infinite way to accomplish said tasks, and when working on a team, those differences can be confusing or hard to manage.
```

Comment:
To make matters worse, there are an infinite number of ways to accomplish said tasks, and managing these differences while working on a team can be confusing and hard to manage.